cypress/support/commands.js

```diff
@@ -0,0 +1,111 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+
+Cypress.Commands.add('getNextSlide', { prevSubject: 'optional' }, () => {
+  return cy.get('.swiper-button-next');
+});
+Cypress.Commands.add('getPrevSlide', { prevSubject: 'optional' }, () => {
+  return cy.get('.swiper-button-prev');
+});
+Cypress.Commands.add('nextSlide', { prevSubject: 'optional' }, () => {
```

Comment:
We shouldn't rely here on some modules, better here will be to use Swiper API and call it as `swiper.slideNext()`. Otherwise we can't slide next if it doesn't have navigation enabled

Reply:
The idea was to rely on swiping, not navigation.
I agree that it should be renamed to a more representative name `navigationNextSlide` maybe?

I think relying on swiping is better than on method `swiper.slideNext()`